templates/serverpod_templates/modulename_server/test/integration/module_endpoint_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:test/test.dart';
+
+// Import the generated test helper file, it contains everything you need.
+import 'test_tools/serverpod_test_tools.dart';
+
+void main() {
+  // This is an example test that uses the `withServerpod` test helper.
+  // `withServerpod` enables you to call your endpoints directly from the test like regular functions.
```

Comment:
Suggestion(NIT): Linebreak at 80 chars.

Reply:
Sure! Seems like a missing linter rule? 😊

Reply:
d8be6ff

Reply:
True, we should probably add something for that!